content/docs/CLI/source-git/init.md

```diff
@@ -1,94 +1,124 @@
 ---
-title: "init"
+title: "packit source-git init"
```

Comment:
The title is shown in the left side menu and with your change, we'd see

```
User Documentation
|- Packit CLI
   |- source-git
      |- packit source-git init
```
which is a bit redundant

Reply:
ah, so I revert it back to just init